stackdriver-debugger/src/com/google/cloud/tools/intellij/stackdriver/debugger/CloudBreakpointHandler.java

```diff
@@ -353,13 +354,15 @@ public void registerBreakpoint(
     }
 
     PsiFile javaFile = psiManager.findFile(ideBreakpoint.getSourcePosition().getFile());
```

Comment:
Since this is no longer a java file only, should we name it a bit more broad?

Reply:
definitely